website/src/libs/api-v3/baseModel.js

```diff
@@ -0,0 +1,53 @@
+import _ from 'lodash';
+import { uuid } from '../../../../common';
+import validator from 'validator';
+
+export default function baseModel (schema, options = {}) {
+  schema.add({
+    _id: {
+      type: String,
+      default: uuid.v4,
+      validate: [validator.isUUID, 'Invalid uuid.'], // TODO check for UUID version
+    },
+  });
+
+  if (options.timestamps) {
+    schema.add({
```

Comment:
Would it be beneficial to have this on every model? Ids there a case where we wouldn't want it?